ddev/src/ddev/cli/meta/scripts/scripts/serve.py

```diff
@@ -11,21 +11,30 @@
 else:
     CONTENT_TYPE = 'plain/text; charset=utf-8'
 
-METRICS_FILE = "/tmp/metrics.txt"
+PORT = 8080
 
 
 class OpenMetricsHandler(BaseHTTPRequestHandler):
     def do_GET(self):
+        global current_payload
+        global payloads
+
+        if len(payloads) > 1:
+            print(f"Serving file {payloads[current_payload]}")
+
         self.send_response(200)
         self.send_header('Content-Type', CONTENT_TYPE)
         self.end_headers()
-        with open(METRICS_FILE, 'rb') as f:
+        with open(f"/tmp/{payloads[current_payload]}", 'rb') as f:
             self.wfile.write(f.read())
 
-
-PORT = 8080
+        # Otherwise we keep using the last one
+        if current_payload < len(payloads) - 1:
+            current_payload += 1
 
 
 if __name__ == '__main__':
+    current_payload = 0
+    payloads = sys.argv[1:]
```

Comment:
hehe, should we consider introducing click here?

Reply:
I'd avoid to do so so we do not add extra deps to the container. Maybe that will make sense in the future if we keep adding new stuff to this, to the point that it would be even better to run the server in a separate container